File: James/230105_study/프로_42842_카펫.java
class Solution {
    public int[] solution(int brown, int yellow) {
        int[] answer = new int[2];
        
        for(int i = 1; i <= brown; i++) {
            for(int j = 1; j <= brown; j++) {
                if((i * 2 + j * 2 + 4) == brown && i >= j && i * j == yellow) {
                    answer[0] = i + 2;
                    answer[1] = j + 2;
                }
            }
        }
        
        return answer;
    }
}

File: Lucas/230105_study/프로_42842_카펫.py
def solution(brown, yellow):
    answer = []
    n,m = 3,3

    while 1:
        if(brown==((n*2 + m*2)-4) and (n*m-((n*2 + m*2)-4))==yellow):
            break
        n += 1 
        if(n+m>2502):
            m += 1
            n = 0
    answer = [n,m]
    return answer

brown = 10
yellow = 2

print(solution(brown,yellow))

